### ISSUE TITLE
Initial code doesn't work

### ISSUE THREAD

**Issue by christopherdent** [learn-co-curriculum/review-map-state-to-props-readme#33]
As someone else pointed out several months ago, the initial code on this lab does not work.  The readme states that 

"So now boot up the app and click on the two buttons. You will see that clicking on the Items Count button renders an update to our App Component, while clicking on the Users Count button does not. This makes sense: inside our App component all we do is reference the items count."

But actually clicking either button results in "TypeError: Cannot read property 'dispatch' of undefined."  

Hard to follow a code-along that is not functioning as the Readme says it should, please fix.  As I said I see another student raised this issue months ago and it looks like no action has been taken.

**Comment by mkirby** [comment]
This issue still persists in the deployment to canvas.

**Comment by mkirby** [comment]
This [commit](https://github.com/learn-co-curriculum/review-map-state-to-props-readme/commit/79fea5c6cf569e5b8326ac32a90b4851778f4a00#diff-bfe9874d239014961b1ae4e89875a6155667db834a410aaaa2ebe3cf89820556) from Jun 18 2020 removed a line of code that passed store into the app component within the index.js file.

**Comment by lizbur10** [comment]
Thank you for spotting this issue and providing feedback.

We have updated the materials and believe your issue to have been resolved. 

We apologize for any frustration you might have encountered during this process but thank you for helping us ensure that those who follow in your path will not encounter the same problems that you faced.

If you do not believe that this issue has been addressed, please re-open this issue. 💙
